runtime/test/description-generator-tests.js

```diff
@@ -20,7 +20,7 @@ var Bar = runtime.testing.testEntityClass('Bar');
 var Far = runtime.testing.testEntityClass('Far');
 
 function testRelevance() {
-  const slotComposer = new SlotComposer({});
+  const slotComposer = new SlotComposer(/* rootContext= */ 'test', /* affordance= */ 'test');
```

Comment:
would it make sense to construct SlotComposer around a dictionary? Then these comments can be part of the source:
const slotComposer = new SlotComposer({rootContext: 'test', affordance: 'test'});